extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -41,24 +41,26 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		return true;
 	}
 
-	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $out ) {
+	public static function onMakeGlobalVariablesScript( array &$vars/*, OutputPage $out*/ ) {
 		$wg = F::app()->wg;
 		$title = $wg->Title->getPrefixedDBkey();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)